iati/tests/events_functional_tests.py

```diff
@@ -215,6 +215,44 @@ def test_event_type_filter(self, admin_browser):
         admin_browser.click_link_by_text(TEST_CATEGORY)
         assert admin_browser.is_text_present("Show all events")
 
+    # Uncomment once we merge editable-header-image
```

Comment:
Minor point - typically things such as this tend to be prepended with `TODO` - that way we can easily search through the repo at any point to find leftover tasks.